### PR TITLE
fix: `patchDevImages.sh`  register arg not take effect

### PR DESCRIPTION
**Pull request by xinnjie**
I tried to change docker registery by add `-r` option, but it not take effect. After some debug, I change `if [ -n "${registry+}" ]` to `if [ -n "${registry}" ]`, then `-r` option take effect.

It looks like a typo

### CODE REVIEW

File: hack/patchDevImages.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ for val in {0..5}; do
     image_names=(webhooks keeper foghorn gc-jobs tekton-controller jenkins-controller)
 
     full_image_name=""
-    if [ -n "${registry+}" ]
+    if [ -n "${registry}" ]
```
Comment (ankitm123) [inline hack/patchDevImages.sh:71]: It should work as it's bash parameter expansion (`registry+` is a shorthand for `registry:+`) See: https://wiki.bash-hackers.org/syntax/pe
May be the error is somewhere else, let me check ...

Comment (xinnjie) [reply hack/patchDevImages.sh:71]: If `registry` is set, then `${registry+}` would be empty, `if` statement would false, and `registry` would not add to `full_image_name`.
So I still think `+` is not needed.

**Pull request closed without merging**